sys/Makefile.dep

```diff
@@ -830,6 +830,7 @@ endif
 
 ifneq (,$(filter suit_transport_%, $(USEMODULE)))
   USEMODULE += suit_transport
+  USEMODULE += suit_transport_worker
```

Comment:
Why worker? Bikeshedding a bit, but how about `common`?

Reply:
I thought `common` first too, but it really only contains code for the CoAP worker thread  - it is entirely optional if you call `suit_parse()` from another thread.

Reply:
Ok